.eslintrc.yml

```diff
@@ -0,0 +1,20 @@
+plugins:
```

Comment:
@charlex Is it possible to only use one eslint config for both the JavaScript and TypeScript files?

Currently, there is an eslintConfig in the root package.json as well and I'd rather one. 

Preference:

- Keep config in the package.json to prevent an overload of config files in the root directory.  
- If a root file is required then use a js file instead `.eslintrc.js`